src/app/components/sprint-detail/sprint-detail.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { DebugElement } from '@angular/core';
+
+import { SprintDetailComponent } from './sprint-detail.component';
+import {ActivatedRoute, Router} from "@angular/router";
```

Comment:
Lets keep consistency of the code and go with { something } or {something}.
Please choose, and lets keep it through the project. And fix it everywhere in the commit.